test/Core.Test/Services/PolicyServiceTests.cs

```diff
@@ -97,6 +98,66 @@ await sutProvider.GetDependency<IEventService>()
                 .LogPolicyEventAsync(default, default, default);
         }
 
+        [Theory, CustomAutoData(typeof(SutProviderCustomization))]
+        public async Task SaveAsync_SingleOrg_VaultTimeoutEnabled_ThrowsBadRequest([PolicyFixtures.Policy(Enums.PolicyType.SingleOrg)] Core.Models.Table.Policy policy, SutProvider<PolicyService> sutProvider)
```

Comment:
Move into its own PR with the new check in PolicyService, along with the other VaultTimeoutPolicy test below.

Reply:
Moved to https://github.com/bitwarden/server/pull/1694